Cargo.toml

```diff
@@ -14,6 +14,13 @@ Timeouts for futures.
 
 [dependencies]
 
+[target.wasm32-unknown-unknown.dependencies]
+instant = { version = "0.1.2", features = ["wasm-bindgen"] }
```

Comment:
I'm wondering if instead we could rely on https://github.com/tomaka/wasm-timer instead? That library seems better maintained, and is probably easier to export directly.

Reply:
That is where these changes came from originally lol. @tomaka (he's away atm) said that it would be better to integrate the changes into something like this crate. I think it's fair to be sceptical of `instant` though. Perhaps the best idea is to remove all the futures stuff from `wasm-timer` and keep `Instant` and `SystemTime`, then we can stabilise the crate.